python/ray/data/dataset.py

```diff
@@ -924,6 +925,28 @@ def mean(self, on: AggregateOnT = None) -> U:
         else:
             return ret[0]
 
+    def std(self, on: AggregateOnT = None, ddof: int = 1) -> U:
+        """Compute standard deviation over entire dataset.
+
+        Examples:
+            >>> ray.data.range(100).std()
+            >>> ray.data.range_arrow(100).std("value")
+
+        Args:
+            on: The data on which to compute the standard deviation.
+                It can be the column name for Arrow dataset.
+            ddof: Delta Degrees of Freedom. The divisor used in calculations
+                is N - ddof, where N represents the number of elements.
```

Comment:
Is this common?

Reply:
Yes this is standard (ha) for standard deviation APIs:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.std.html
https://numpy.org/doc/stable/reference/generated/numpy.std.html